rb_ws/src/buggy/scripts/auton/autonsystem.py

```diff
@@ -115,18 +116,33 @@ def update_other_steering_angle(self, msg):
         with self.lock:
             self.other_steering = msg.data
 
+    def update_rtk_status(self, msg):
+        with self.lock:
+            self.rtk_status = msg.data
+
+    def init_check(self):
+        # checks that messages are being receieved
+        # (from both buggies if relevant),
+        # RTK status is fixed
+        # covariance is less than 1 meter
+        if (self.self_odom_msg == None) or (self.has_other_buggy and self.other_odom_msg == None) or (self.rtk_status <= 5) or (self.self_odom_msg.pose.covariance[0] ** 2 + self.self_odom_msg.pose.covariance[7] ** 2 > 100):
```

Comment:
According to the old code, covariance is meters. This is checking for 10m covariance